mpas_analysis/shared/plot/plotting.py

```diff
@@ -98,39 +298,7 @@ def timeseries_analysis_plot(config, dsvalues, N, title, xlabel, ylabel,
                  lineStyles[dsIndex],
                  linewidth=lineWidths[dsIndex])
 
-    ax = plt.gca()
-
-    # Add a y=0 line if y ranges between positive and negative values
-    yaxLimits = ax.get_ylim()
-    if yaxLimits[0]*yaxLimits[1] < 0:
-        indgood = np.where(np.logical_not(np.isnan(mean)))
-        x = mean['Time'][indgood]
-        plt.plot(x, np.zeros(np.size(x)), 'k-', linewidth=1.2)
```

Comment:
the 6 lines above are now missing. Not sure where they should go. Perhaps before calling `_plot_xtick_format`?
They print a y=0 line when the y-axis spans negative and positive values, and it seemed useful for reading OHC time series in particular.